### PR TITLE
Fix typos in comments and docs

### PR DESCRIPTION
**Pull request by eebasso**
This PR fixes some spelling, grammar, and other typos in the code comments and documentation.

### CODE REVIEW

File: Source/FieldSolver/FiniteDifferenceSolver/FiniteDifferenceAlgorithms/CylindricalYeeAlgorithm.H [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
 /**
  * This struct contains only static functions to initialize the stencil coefficients
- * and to compute finite-difference derivatives for the Cartesian Yee algorithm.
+ * and to compute finite-difference derivatives for the Cylindrial Yee algorithm.
```
Comment (roelof-groenewald) [inline Source/FieldSolver/FiniteDifferenceSolver/FiniteDifferenceAlgorithms/CylindricalYeeAlgorithm.H]:
```suggestion
 * and to compute finite-difference derivatives for the Cylindrical Yee algorithm.
```

File: Docs/source/install/hpc/karolina.rst [hunk 2]
```diff
@@ -17,7 +17,7 @@ If you are new to this system, **please see the following resources**:
 * `Filesystems <https://docs.it4i.cz/karolina/storage/>`__:
 
   * ``$HOME``: per-user directory, use only for inputs, source and scripts; backed up (25GB default quota)
-  * ``/scatch/``: `production directory <https://docs.it4i.cz/karolina/storage/#scratch-file-system>`__; very fast for parallel jobs (20TB default)
+  * ``/scratch/``: `production directory <https://docs.it4i.cz/karolina/storage/#scratch-file-system>`__; very fast for parallel jobs (20TB default)
```
Comment (ax3l) [inline Docs/source/install/hpc/karolina.rst:20]: cc @berceanu fyi - probably a typo?

**Review by ax3l — APPROVED** [review]
Thank you for the fixes :sparkles:

**Comment by ax3l** [comment]
@eebasso there is a small merge conflict from another PR in `Docs/source/usage/examples.rst` now - can you please rebase?

**Comment by eebasso** [comment]
> @eebasso there is a small merge conflict from another PR in `Docs/source/usage/examples.rst` now - can you please rebase?

Should be done now.

**Pull request merged**